tests.js

```diff
@@ -114,3 +132,39 @@ Tinytest.add('template-extension - parentData', function (test) {
     testingData = false;
   }
 });
+
+Tinytest.add('template-extension - inheritsHelpersFrom', function (test) {
+  Template.testTemplate3.inheritsHelpersFrom('testTemplate2');
+  test.equal(Template.testTemplate2.__helpers[' testInstanceGet'], Template.testTemplate3.__helpers[' testInstanceGet']);
+});
+
+Tinytest.add('template-extension - inheritsHelpersFrom array', function (test) {
+  Template.testTemplate4.inheritsHelpersFrom(['testTemplate1', 'testTemplate2']);
+  test.equal(Template.testTemplate2.__helpers[' testInstanceGet'], Template.testTemplate4.__helpers[' testInstanceGet']);
+  test.equal(Template.testTemplate1.__helpers[' data'], Template.testTemplate4.__helpers[' data']);
+});
+
+Tinytest.add('template-extension - inheritsEventsFrom', function (test) {
+  Template.testTemplate5.inheritsEventsFrom('testTemplate3');
+  test.equal(Template.testTemplate3.__eventMaps, Template.testTemplate5.__eventMaps);
+});
+
+Tinytest.add('template-extension - inheritsEventsFrom array', function (test) {
+  Template.testTemplate6.inheritsEventsFrom(['testTemplate3', 'testTemplate4']);
+  test.equal(Template.testTemplate3.__eventMaps[0], Template.testTemplate6.__eventMaps[0]);
+  test.equal(Template.testTemplate4.__eventMaps[0], Template.testTemplate6.__eventMaps[1]);
+});
+
+Tinytest.add('template-extension - inheritsHooksFrom', function (test) {
+  Template.testTemplate7.inheritsHooksFrom('testTemplate');
+  Template.testTemplate7.created();
+  test.equal(Template.testTemplate7._testTemplateField, 42);
+});
+
+Tinytest.add('template-extension - inheritsHooksFrom array', function (test) {
+  Template.testTemplate7.inheritsHooksFrom(['testTemplate', 'testTemplate8']);
+  Template.testTemplate7.created();
+  Template.testTemplate7.rendered();
+  test.equal(Template.testTemplate7._testTemplateField, 42);
+  test.equal(Template.testTemplate7._testTemplateField4, 14);
```

Comment:
This test function should probably use something other than `testTemplate7`. The synchronous tests run in order, meaning you've already called `Template.testTemplate7.created()` in the previous test, so `_testTemplateField` is already set when this test begins. You could either clear the hooks from `testTemplate7` at the beginning of this test, or just use a new blank template.